python/lsst/ip/isr/isrTask.py

```diff
@@ -959,6 +960,8 @@ def readIsrData(self, dataRef, rawExposure):
         # immediate=True required for functors and linearizers are functors; see ticket DM-6515
         linearizer = (dataRef.get("linearizer", immediate=True)
                       if self.doLinearize(ccd) else None)
+        if linearizer is not None and not isinstance(linearizer, numpy.ndarray):
```

Comment:
Is this check for ndarray a backwards compatibility thing?

Reply:
Yes.  Until we move everything to consistent calibration types, we don't know what we got from the butler.